tests/data/pluto_disk_planet/meta.yaml

```diff
@@ -5,6 +5,7 @@ attrs:
   geometry: spherical
   dimensionality: 3
   kind: pluto
+  def_units: True
```

Comment:
Seems like a good solution, but I'm not sure the variable name is clear enough. If you mean to indicate that units can be parsed from `definitions.h`, please rename this key to `"has_units"` to follow the same naming convention as `"has_geometry"`.
Also it'd be good to add this parameter to idefix datasets `meta.yaml` files (you can set it to False every where there)